Sources/Jobs/ScheduledJobStorage.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ScheduledJobStorage.swift
+//  
+//
+//  Created by Jimmy McDermott on 6/11/19.
+//
+
+import Foundation
+import Vapor
+
+struct ScheduledJobStorage {
+    let scheduledJob: ScheduledJob
+    let scheduler: Scheduler
```

Comment:
The scheduler is a reference type, so we probably shouldn't store it inside of a struct since that makes copy-on-write semantics kind of weird.

The scheduler being a reference is useful for creating a fluent building api, but we should probably store the information in a struct. 

On a tangent, Swift 5.1's function builder's might be perfect for this...

Reply:
Hmm... storing the Scheduler info in a struct wouldn't work if we want to be able to chain on the methods after calling `schedule` on the configuration object. It would need to be a reference type so that we have access to the modifications made by the end user. Lmk if you end up coming up with a solution you like better during your refactoring.